### ISSUE TITLE
Missing dependency

### ISSUE THREAD

**Issue by marcverhagen** [clamsproject/mmif-python#199]
When you just need the Python MMIF SDK you would like to do simple install with `pip install mmif-python` and then just import it, but that gets you get an import error:

```
ModuleNotFoundError: No module named 'jsonschema'
```

So you need to do an extra install. This is not an issue with clams-python which has the jsonschema dependency listed in its requirements file. Do the same here.

**Comment by marcverhagen** [comment]
It may also be a good idea to add `lapps==0.0.2` here and remove it from the clams-python dependencies.

**Comment by keighrim** [comment]
9212994 adds `jsonschema` in requirements.txt. `lapps` isn't actually required at the moment, so I didn't include it. 

``` bash 
$ shopt -s globstar

$ grep import **/*.py | grep schema
dist/mmif-python-0.4.7.dev1/mmif/serialize/mmif.py:import jsonschema.validators
mmif/serialize/mmif.py:import jsonschema.validators
tests/test_serialize.py:import hypothesis_jsonschema  # pip install hypothesis-jsonschema
tests/test_serialize.py:from jsonschema import ValidationError

$ grep import **/*.py | grep lapps

$ 
```

**Comment by keighrim** [comment]
Closing as all dependencies were updated via 9212994 and [released as 0.4.7](https://github.com/clamsproject/mmif-python/pull/203).
